gui-wm/sway/sway-9999.ebuild

```diff
@@ -34,23 +34,19 @@ DEPEND="
 	x11-libs/pixman
 	media-libs/mesa[gles2,libglvnd(+)]
 	swaybar? ( x11-libs/gdk-pixbuf:2 )
-	swaybg? ( gui-apps/swaybg )
-	swayidle? ( gui-apps/swayidle )
-	swaylock? ( gui-apps/swaylock )
 	tray? ( || (
 		sys-apps/systemd
 		sys-auth/elogind
 		sys-libs/basu
 	) )
-	wallpapers? ( x11-libs/gdk-pixbuf:2[jpeg] )
```

Comment:
I think this is still needed for the following type of line in sway config

`output * bg ~/.wallpapers/foobar.jpg fill`

Reply:
Well, kinda not really.

Sway uses per default `swaybg` for printing a wallpaper. `swaybg` however isn't part of the `gui-wm/sway` package and comes with `gui-apps/swaygb`.
It's correct that we need `x11-libs/gdk-pixbuf:2[jpeg]` in order to set jpg wallpapers, however it's only required for `swaybg` which already enables the dependency via the `gdk-pixbuf` use flag.

If we want to control it in the `gui-wm/sway` ebuild we would need to set it as following:
```
wallpapers? ( gui-apps/swaybg[gdk-pixbuf] )
```

However, since there are other wallpaper tools i would let the user choose which one to use.

Reply:
`option('gdk-pixbuf', type: 'feature', value: 'auto', description: 'Enable support for more image formats in swaybg')`

maybe that should either be set statically or behind the wallpaper flag, or maybe it's a defunct flag (it's 4 years old now).

I think the output bg option only calls swaybg and given that the example config has a section where it calls swaybg I'd say we should default enable a swaybg or wallpaper flag and depend on gdk-pixbuf as you have in the comment here.

Reply:
I guess it's simply a leftover from when `swaybg` was split into it's own project.

BTW, you can set a different wallpaper tool via the `swaybg_command` setting.
However i'm fine with with adding the `swaybg` dependency as in my example. I'll add it.